# Example for an AR Inspection task
---
slides:
    
  - instruction: "Look at the three points"
    boxes:
      - frame_id: "table_board_center"
        position: {x: 0.0, y: 0.0, z: 0.0}
        width: 0.50
        height: 0.20
        border_color: {r: 0.0, g: 0.0, b: 1.0, a: 1.0}
        fill_color: {r: 0.0, g: 0.0, b: 0.0, a: 0.0}
        label: "The Box"
    pois:
      - frame_id: "table_board_center"
        position: {x: 0.0, y: -0.15, z: 0.0}
        radius: 0.01
        border_color: {r: 1.0, g: 0.0, b: 0.0, a: 1.0}
        fill_color: {r: 1.0, g: 1.0, b: 1.0, a: 1.0}
        label: "top"
      - frame_id: "table_board_center"
        position: {x: 0.0, y: -0.0, z: 0.0}
        radius: 0.02
        border_color: {r: 1.0, g: 0.0, b: 0.0, a: 1.0}
        fill_color: {r: 1.0, g: 1.0, b: 1.0, a: 1.0}
        label: "center"
      - frame_id: "table_board_center"
        position: {x: 0.0, y: 0.15, z: 0.0}
        radius: 0.04
        border_color: {r: 1.0, g: 0.0, b: 0.0, a: 1.0}
        fill_color: {r: 1.0, g: 1.0, b: 1.0, a: 1.0}
        label: "bottom"

  - instruction: "Look at the three points"
    boxes:
      - frame_id: "table_board_center"
        position: {x: 0.0, y: 0.0, z: 0.0}
        width: 0.20
        height: 0.50
        border_color: {r: 0.0, g: 0.0, b: 1.0, a: 1.0}
        fill_color: {r: 0.0, g: 0.0, b: 0.0, a: 0.0}
        label: "The Box"
    pois:
      - frame_id: "table_board_center"
        position: {x: -0.15, y: 0, z: 0.0}
        radius: 0.01
        border_color: {r: 1.0, g: 0.0, b: 0.0, a: 1.0}
        fill_color: {r: 1.0, g: 1.0, b: 1.0, a: 1.0}
        label: "left"
      - frame_id: "table_board_center"
        position: {x: 0.0, y: 0.0, z: 0.0}
        radius: 0.02
        border_color: {r: 1.0, g: 0.0, b: 0.0, a: 1.0}
        fill_color: {r: 1.0, g: 1.0, b: 1.0, a: 1.0}
        label: "center"
      - frame_id: "table_board_center"
        position: {x: 0.15, y: 0.0, z: 0.0}
        radius: 0.04
        border_color: {r: 1.0, g: 0.0, b: 0.0, a: 1.0}
        fill_color: {r: 1.0, g: 1.0, b: 1.0, a: 1.0}
        label: "right"